api/routers/legacy.py

```diff
@@ -1112,11 +1131,15 @@ async def post_verification_request_information(token: str, verify_info: Discord
     discord_id = info.get("discord_id")
     code = info.get("code")
 
+    assert(player_name is not None)
```

Comment:
The assert keyword is used when debugging code.
The assert keyword lets you test if a condition in your code returns True, if not, the program will raise an AssertionError.
a simple if None== name or None == x or None == y: logger.debug() raise httpexception